### PR TITLE
Remove references to LONG_LONG_MAX

### PR DESCRIPTION
**Pull request by deadalnix**
This is extracted from the C++11 failed patch. this part is relatively harmless, so should be merged already.

### CODE REVIEW

File: src/leakybucket.h [hunk 1]
```diff
@@ -52,7 +52,7 @@ class CLeakyBucket
     }
 
 public:
-    CLeakyBucket(int64_t maxp, int64_t fillp, int64_t startLevel = LONG_LONG_MAX) : max(maxp), fill(fillp)
+    CLeakyBucket(int64_t maxp, int64_t fillp, int64_t startLevel = std::numeric_limits<long long>::max()) : max(maxp), fill(fillp)
```
Comment (deadalnix) [inline src/leakybucket.h:55]: This won't work without const expr. Fixing.

**Review by deadalnix — COMMENTED** [review]
Found out the problem, fixing.

**Comment by gandrewstone** [comment]
I don't see how this PR could have caused the travis hang that was shown... restarting the jobs...

**Comment by sickpig** [comment]
fixes #38

**Pull request merged**